### DISCUSSION TITLE
Few folder QoL additions

### DISCUSSION THREAD

**Discussion by ItsMelodyy0102** [diced/zipline#574] [Ideas]
I'd very much love to see the option to allow bulk adding/removing files to folders. Some sort of multi-select system + allowing to add multiple files to folders, delete, etc. would be fantastic.

Additionally, it'd be awesome if we had an option either via a docker environmental or as config in the manage account page, if you are able to show folder name in the URL instead of the ID (e.g domain.com/folder/derp instead of domain.com/folder/5)

**Comment by diced** [comment]
> I'd very much love to see the option to allow bulk adding/removing files to folders. Some sort of multi-select system + allowing to add multiple files to folders, delete, etc. would be fantastic.

This has been added into v4 already!

> it'd be awesome if we had an option either via a docker environmental or as config in the manage account page

It's a bit complicated, but I have been experimenting with ways to move some of the less important configuration options out of an environment variable. I don't really know if i will go ahead with it though.

> if you are able to show folder name in the URL instead of the ID (e.g domain.com/folder/derp instead of domain.com/folder/5)

I think a better way to handle this is just allowing both the ID and the name of the folder to be accessible through the URL, I think I may add that soon.

**Reply by ItsMelodyy0102** [reply]
Awesome! What about the ability to rename folders though? Is that also added to V4 already, or is that something I can add to this discussion too?
